File: kotlin-java-codingtest/src/main/kotlin/programmers/lv0/날짜_비교하기/날짜 비교하기.kt
package programmers.lv0.날짜_비교하기

import java.time.LocalDate


fun solution(date1: IntArray, date2: IntArray): Int {
    val d1 = LocalDate.of(date1[0], date1[1], date1[2])
    val d2 = LocalDate.of(date2[0], date2[1], date2[2])

    return if (d1.isBefore(d2)) 1 else 0
}

File: Python/programmers/lv0/날짜 비교하기.py
def solution(date1, date2):
    if date1[0] < date2[0]:
        return 11
    elif date1[0] > date2[0]:
        return 0
    else:
        if date1[1] < date2[1]:
            return 1
        elif date1[1] > date2[1]:
            return 0
        else:
            if date1[2] < date2[2]:
                return 1
            elif date1[2] >= date2[2]:
                return 0
